sdk/src/types/block/output/feature/mod.rs

```diff
@@ -37,6 +37,7 @@ pub enum Feature {
 }
 
 impl PartialOrd for Feature {
+    #[allow(clippy::incorrect_partial_ord_impl_on_ord_type)]
```

Comment:
Or fix it?

Reply:
But what it suggests would actually be wrong, no?

Reply:
no?

Reply:
It suggested `Some(self.cmp(other))`, but `cmp()` calls `partial_cmp()`

Reply:
So make it not?

Reply:
But that would change `cmp()` then

Reply:
Yes, you should flip the impls

Reply:
But that's different behavior then and could break something, no?

Reply:
Yes now I see it as well, somehow I thought `cmp()` would compare also the content and not only the `kind()`

Reply:
Done